app/views/reviews/new/_facility_buttons.html.erb

```diff
@@ -0,0 +1,39 @@
+<% # Locals:
+   # - facility (one Facility)
+   # - form (Form for a new or editing Review)
+   # - facility_review (a new or old facility)
+%>
+<%
+  icon = facility.icon
```

Comment:
Kanskje putte inn i en hjelper?

Reply:
Kan bare fjerne dem. De hjelper ikke koden noe, for begge deler (både icon og title) brukes bare en gang.